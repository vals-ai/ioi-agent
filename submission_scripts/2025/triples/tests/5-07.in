version https://git-lfs.github.com/spec/v1
oid sha256:8fbe4a7f07096a9c9bfa15c82ed465a9ee5ae7a2d284c2e04fb62d83d7c29c0f
size 255770
