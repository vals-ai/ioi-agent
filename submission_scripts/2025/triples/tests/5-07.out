version https://git-lfs.github.com/spec/v1
oid sha256:ebf5f4a4421daa53742328366c2329c031a358d899f709ab73436221d8a5299c
size 39
